### ISSUE TITLE
New API Request: get order by userid

### ISSUE THREAD

**Issue by YanB25** [wwyf/Waterfall-backend#16]
Add an api `/api/user/orders` that return all **his** mainOrders(subOrders) if he is customer(provider).

I suggest to use the format
```
{
    code: 0,
    data: {
        order: [{
             ...
        },]
    }
}
``` 
since `order` would be unambiguous both in frontend and backend.

**Comment by gtbl2012** [comment]
consider change the api response to the one below?
`{
    code: 0,
    data: {
        mainOrder: [{
             ...
        },],
        subOrder: [{
             ...
        },]
    }
}`

**Comment by YanB25** [comment]
That's okay.
If I am provider, is `mainOrder` field empty(`[]`)?

**Comment by wwyf** [comment]
fixed by #27
